ui/client/src/views/Keys.tsx

```diff
@@ -0,0 +1,368 @@
+// Copyright © 2025 Kaleido, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Alert, Box, Breadcrumbs, Button, Fade, IconButton, Link, Paper, Table, TableBody, TableCell, TableContainer, TableHead, TablePagination, TableRow, TableSortLabel, Tooltip, Typography } from "@mui/material";
+import { useQuery } from "@tanstack/react-query";
+import { t } from "i18next";
+import { useEffect, useState } from "react";
+import { fetchKeys } from "../queries/keys";
+import { Hash } from "../components/Hash";
+import FolderOpenIcon from '@mui/icons-material/FolderOpen';
+import NavigateNextIcon from '@mui/icons-material/NavigateNext';
+import { IKeyEntry, IVerifier } from "../interfaces";
+import { useSearchParams } from "react-router-dom";
+import { Captions, Signature } from "lucide-react";
+import { constants } from "../components/config";
+import SearchIcon from '@mui/icons-material/Search';
+import { ReverseKeyLookupDialog } from "../dialogs/ReverseKeyLookup";
+import RemoveIcon from '@mui/icons-material/Remove';
+import { VerifiersDialog } from "../dialogs/Verifiers";
+
+export const Keys: React.FC = () => {
+
+  const getDefaultRowsPerPage = () => {
+    const valueFromStorage = window.localStorage.getItem(constants.KEYS_ROWS_PER_PAGE);
```

Comment:
ok - so maybe now I understand why we have separate constants.
We're using separate sections in the storage, to keep the status of each table - so i you nav somewhere else, and then back to the table - we'll remember where you were (not just on the back button, as would be the case with the query strings in the location HREF).

Reply:
That's exactly right. Even if you close the web browser and open it again, we'll "remember" what the user preference was when it comes to what column they want to sort by and what order (asc/desc).